configuration/index.md

```diff
@@ -16,23 +16,25 @@ You as the end-user have the **full control** over every aspect of your smart ho
 Every device connected to openHAB is functionally and logically different.
 In order to represent all of these, openHAB defines the following few base components:
 
-- [Add-ons](addons.html) - The numerous Add-ons to communicate with your devices
+- [Bindings]({{base}}/addons/bindings.html) - The numerous Bindings to communicate with your devices
```

Comment:
I would go  for `add-ons` here and also link to `[Add-ons](/addons/)`
We are using the term add-ons as a new area in our new website.

If you want to jump to bindings specifically `[Bindings](/addons/#binding)` would be the working link.

(It's a bit of a mess while restructuring this, i know...)